pkg/cmd/roachtest/decommission.go

```diff
@@ -462,6 +462,8 @@ func runDecommissionAcceptance(ctx context.Context, t *test, c *cluster) {
 		t.Fatalf("recommission failed: %v", err)
 	}
 
+	// TODO(knz): quit --decommission is deprecated in 20.1. Remove
+	// this roachtest in 20.2.
```

Comment:
just this part of the roachtest, not the roachtest

Reply:
Good point. Fixed.